src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -36,9 +30,9 @@ class Repository(
 
     offender.bookings.addAll(
       offenderBuilder.bookingBuilders.mapIndexed { index, bookingBuilder ->
-        val booking = save(bookingBuilder.build(offender, index, lookupAgency(bookingBuilder.agencyLocationId)))
```

Comment:
removed these unnecessary saves since it will get saved when the offender gets saved anyway